backstop/tests/table.html

```diff
@@ -1250,595 +1727,311 @@ <h2 id="default">Default</h2>
-                <svg
-                  class="iui-icon iui-tables-menu"
-                  viewBox="0 0 16 16"
-                  aria-hidden="true"
-                >
-                  <path
-                    d="m4 8a2 2 0 1 1 -2-2 2 2 0 0 1 2 2zm4-2a2 2 0 1 0 2 2 2 2 0 0 0 -2-2zm6 0a2 2 0 1 0 2 2 2 2 0 0 0 -2-2z"
-                  />
-                </svg>
-              </div>
+                <path
+                  d="M8,3,6.5856,1.5857A2,2,0,0,0,5.1715,1H1A1,1,0,0,0,0,2V5H16V4a1,1,0,0,0-1-1Z"
+                  fill="#e7b93a"
+                />
+                <path
+                  d="M0,14a1,1,0,0,0,1,1H15a1,1,0,0,0,1-1V5H0Z"
+                  fill="#f6d97a"
+                />
+              </svg>
+              Folder name 1
             </div>
-
-            <div class="iui-tables-row iui-tables-row-active">
-              <div class="iui-tables-cell iui-tables-slot">
-                <label class="iui-checkbox">
-                  <input
-                    type="checkbox"
-                    id="active-row"
-                    checked
-                  />
-                  <span class="iui-checkbox-checkmark">
-                    <svg
-                      viewBox="0 0 16 16"
-                      aria-hidden="true"
-                      focusable="false"
-                    >
-                      <path
-                        class="iui-check"
-                        d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z"
-                      />
-                    </svg>
-                  </span>
-                </label>
-              </div>
-              <div class="iui-tables-cell iui-tables-filename">
-                <!-- Use folder.svg from @itwin/itwinui-icons-color package. -->
-                <svg
-                  viewBox="0 0 16 16"
-                  aria-hidden="true"
-                >
-                  <path
-                    d="M8,3,6.5856,1.5857A2,2,0,0,0,5.1715,1H1A1,1,0,0,0,0,2V5H16V4a1,1,0,0,0-1-1Z"
-                    fill="#e7b93a"
-                  />
-                  <path
-                    d="M0,14a1,1,0,0,0,1,1H15a1,1,0,0,0,1-1V5H0Z"
-                    fill="#f6d97a"
-                  />
-                </svg>
-                Folder name 3
-              </div>
-              <div
-                class="iui-tables-cell"
-                title="Jul 19, 2020 4:20pm"
+            <div
+              class="iui-cell"
+              title="Jul 16, 2020 9:50am"
+            >
+              Last week
+            </div>
+            <div class="iui-cell">
+              <user-icon
+                size="small"
+                type="2"
+              >
+              </user-icon>
+              Terry Rivers
+            </div>
+            <div class="iui-cell iui-right-align">252 MB</div>
+            <div class="iui-cell iui-slot">
+              <button
+                class="iui-button iui-borderless iui-more-options"
```

Comment:
I think more-options needs some custom styling for extra-condensed (and maybe even condensed) density. Looks huge especially when compared to the column manager (which is also a borderless button).

Reply:
Oops, that is a mistake.  I've added `.iui-small` to the `.iui-more-options` buttons within condensed & extra condensed tables.